backend/remote-state/s3/backend.go

```diff
@@ -155,20 +155,23 @@ func New() backend.Backend {
 				Optional:    true,
 				Description: "Skip getting the supported EC2 platforms.",
 				Default:     false,
+				Deprecated:  "The S3 Backend does not require EC2 functionality and this attribute is no longer used.",
 			},
 
 			"skip_region_validation": {
 				Type:        schema.TypeBool,
 				Optional:    true,
 				Description: "Skip static validation of region name.",
 				Default:     false,
+				Deprecated:  "This attribute is no longer used.",
 			},
 
 			"skip_requesting_account_id": {
 				Type:        schema.TypeBool,
 				Optional:    true,
 				Description: "Skip requesting the account ID.",
 				Default:     false,
+				Deprecated:  "The S3 Backend no longer automatically uses IAM or STS functionality to lookup the AWS Account ID and this attribute is no longer used.",
```

Comment:
Nitpick: IAM or STS are mostly implementation details, so we could just say "The S3 Backend no longer looks up the AWS Account ID and this attribute is no longer used." ?

Reply:
No problem will update tomorrow :+1: